packages/@react-spectrum/color/docs/ColorField.mdx

```diff
@@ -96,11 +96,41 @@ When the `necessityIndicator` prop is set to `"label"`, a localized string will
 
 ## Step values
 
-The `step` prop can be used to snap the value to certain increments, default is one. The steps are calculated starting from
-zero or the current value. A `step` can be any positive decimal.
+The `step` prop can be used to snap the value to certain increments. The steps are calculated starting from
+zero or the current value. A `step` can be set to any positive decimal, defaulting to `1` if unspecified.
 
 ```tsx example
-<NumberField label="Primary Color" step={16} />
+<ColorField label="Primary Color" step={16} />
+```
+
+## Validation
+
+NumberFields can display a validation state to communicate to the user whether the current value is valid or invalid.
+Implement your own validation logic in your app and pass either `"valid"` or `"invalid"` to the NumberField via the `validationState` prop.
+
+The example below illustrates how one would validate if the user has entered a red color.
+```tsx example
+function Example() {
+  let [value, setValue] = React.useState('#e73623');
+  let isValid = React.useMemo(() => {
+    if (value.length === 7) {
```

Comment:
Noticed that the example breaks if the user deletes everything in the field, think you have to make sure value is defined here too.